app/isotopomer/__init__.py

```diff
@@ -474,7 +473,8 @@ def extract_site_dictionary_from_dash_triggers(states):
         else:
             site["quadrupolar"]["Cq"] *= 1.0e6  # Cq in MHz
             for angle in ["alpha", "beta", "gamma"]:
-                site["quadrupolar"][angle] = math.radians(site["quadrupolar"][angle]) or None
+                if angle in site["quadrupolar"].keys():
+                    site["quadrupolar"][angle] = math.radians(site["quadrupolar"][angle]) or None
```

Comment:
None doesn't have relevance anymore. 
`site["quadrupolar"][angle] = math.radians(site["quadrupolar"][angle])`
should be enough.